modules/turnero/routes/configuracionPantalla.ts

```diff
@@ -120,22 +123,23 @@ router.delete('/pantalla/:id', Auth.authenticate(), async (req: any, res, next)
 
 router.post('/pantalla/activate', async (req, res, next) => {
     let codigo = req.body.codigo;
+    let tipoPantalla = req.body.tipo;
     let pantallas = await TurneroPantalla.find({
         token: codigo,
+        tipo: tipoPantalla,
         expirationTime: { $gt: new Date() },
     });
     if (pantallas.length) {
         let pantalla: any = pantallas[0];
         pantalla.token = null;
         pantalla.expirationTime = null;
         await pantalla.save();
-
-        let token = Auth.generateAppToken(pantalla, {}, [`turnero:${pantalla._id}`], 'turnero-token');
+        let token = (pantalla.tipo === 'totem') ? Auth.generateAppToken(pantalla, {}, [`${pantalla.tipo}:${pantalla._id}`], `totem-token`) : Auth.generateAppToken(pantalla, {}, [`${pantalla.tipo}:${pantalla._id}`], `turnero-token`);
```

Comment:
Detalle para que se entienda más el código.
```suggestion 
        const tokenType = (pantalla.tipo === 'totem') ? `totem-token` : `totem-token`
        const token = Auth.generateAppToken(pantalla, {}, [`${pantalla.tipo}:${pantalla._id}`], tokenType);
```